lib/deployServiceClient.js

```diff
@@ -0,0 +1,152 @@
+const io = require("socket.io-client"),
+  url = require("url"),
+  request = require("request"),
+  fs = require("fs"),
+  logger = require("../lib/kit").logger,
+  mime = require("mime"),
+  readdirp = require("readdirp");
+
+const directoriesToIgnore = ["!.git", "!*modules"];
```

Comment:
@ignacy given that now we have private/public AND modules directories as a preferred way of developing, i dont know if that glob is still correct